conans/test/integration/command/new_test.py

```diff
@@ -36,6 +36,18 @@ def test_new_cmake_exe(self):
         client.run("export . --user=myuser --channel=testing")
         assert "pkg/1.3@myuser/testing" in client.out
 
+    def test_new_missing_definitions(self):
+        client = TestClient()
+        client.run("new cmake_lib", assert_error=True)
+        assert "ERROR: Missing name, version definitions. " \
+               "Required definitions are: name, version" in client.out
+        client.run("new cmake_lib -d name=name", assert_error=True)
+        assert "ERROR: Missing version definitions. " \
```

Comment:
Might be a bit confusing sentence. Probably better to single quote the attributes and use "definition" (singular).
`ERROR: Missing 'version' definition.`